features/support/javascript.rb

```diff
@@ -76,12 +76,19 @@ def select_from_choices(value, options)
     select_has_value_as_text = all(".choices__list.choices__list--single", visible: false, text: value, wait: false).present?
     return if select_has_value_as_text || select.value == value
 
-    choices_option = div.find(
-      "div[role='option'][class='choices__item choices__item--choice choices__item--selectable']",
-      text: value,
-      visible: false,
-    )
-
+    choices_option = if options[:from] == "edition_publication_type_id"
```

Comment:
I wonder if instead of adding a special case for `edition_publication_type_id` we try `role='option'` first and if that's not found change to `role='treeitem'`?
What do you think?